src/Chullo.php

```diff
@@ -322,7 +323,7 @@ public function copyResource(
      * Issues a MOVE request to Fedora.
      *
      * @param string    $uri            Resource URI
-     * @param array     $destination    Destination URI
+     * @param string    $destination    Destination URI
```

Comment:
Also, this change should probably be continued to FedoraApi's `moveResource`. We probably also need to look at `prepareUri`, it also seems to be a string but the doc comment says array (again both in Chullo and FedoraApi)